doc/api/deprecations.md

```diff
@@ -660,6 +660,28 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+### DEP00XX: tls.parseCertString()
+
+Type: Documentation-only
+
+`tls.parseCertString()` is a trivial parsing helper that was made public by
```

Comment:
Nit: It would be better if we could provide a link here.

Reply:
What link? 🤔